src/components/editor/Bindings/SchemaInference/Dialog/CancelButton.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button } from '@mui/material';
+import { useBindingsEditorStore_setInferredSchemaApplicationErrored } from 'components/editor/Bindings/Store/hooks';
+import {
+    secondaryButtonBackground,
+    secondaryButtonHoverBackground,
+} from 'context/Theme';
+import { Dispatch, SetStateAction } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+interface Props {
+    setOpen: Dispatch<SetStateAction<boolean>>;
+}
+
+function CancelButton({ setOpen }: Props) {
+    // Bindings Editor Store
+    const setInferredSchemaApplicationErrored =
+        useBindingsEditorStore_setInferredSchemaApplicationErrored();
+
+    const closeConfirmationDialog = (event: React.MouseEvent<HTMLElement>) => {
+        event.preventDefault();
+
+        setOpen(false);
+
+        setInferredSchemaApplicationErrored(false);
+    };
+
+    return (
+        <Button
+            onClick={closeConfirmationDialog}
+            sx={{
+                'backgroundColor': (theme) =>
+                    secondaryButtonBackground[theme.palette.mode],
+                '&:hover': {
+                    backgroundColor: (theme) =>
+                        secondaryButtonHoverBackground[theme.palette.mode],
+                },
```

Comment:
Thinking

I wonder if we could handle something like this in the theme itself as we often want both the bgColor and the hovered bgColor.